tests/infra/ccf.py

```diff
@@ -150,7 +150,7 @@ def start_and_join(self, args):
 
         self.nodes_json()
         self.add_members([1, 2, 3])
-        self.add_users([1, 2, 3])
+        self.add_users([1, 2, 3, 4, 5, 6])
```

Comment:
This is probably a bit of an issue as it will pollute the `build/` directory with public keys/certs for _all_ end-to-end tests. We could perhaps add a new argument to `start_and_join()` to define how many users we want?

Reply:
ah good catch! If that is a thing we want to support it should probably go in a separate PR. For this one I will reduce the number of users to 3